test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -842,6 +842,14 @@ add_specs suite_builder =
             Math.min (Decimal.new "12E70") (Decimal.new "13E70") . should_equal (Decimal.new "12E70")
             Math.max (Decimal.new "12E70") (Decimal.new "13E70") . should_equal (Decimal.new "13E70")
 
+    suite_builder.group "BigDecimal internal representation methods" group_builder->
+        group_builder.specify "internal accessors should work correctly" <|
+            d = Decimal.new "123.456"
+            d.precision . should_equal 6
+            d.scale . should_equal 3
+            d.unscaled_value . should_equal 123456
+            d.internal_representation . should_equal [123456, 6, 3]
```

Comment:
Why do we need these 2 tests?

Reply:
There are no other direct tests of these methods.